docs/source/howto/installation.rst

```diff
@@ -558,13 +558,13 @@ In order to restore a backup, you will need to:
 
 Managing multiple users
 =======================
-Setups with multiple users for a single AiiDA instance are currently not supported.
-Instead, each AiiDA user should install AiiDA in a Unix/Windows account on their own computer.
-Under this account it is possible to configure all the credentials necessary to connect to remote computers.
-Using independent accounts will ensure that, for instance, the SSH credentials to connect to supercomputers are not shared with others.
+AiiDA currently does not support multiple users running concurrently on the same AiiDA instance (database & file repository).
+While AiiDA will tag any node with the :py:class:`~aiida.orm.users.User` who created it, there is currently no permission system in place to limit the operations that can be performed by a given user.
```

Comment:
This would maybe benefit from first explaining that each database can hold a number of `User`s, and the profile designates one of these as the default user. All nodes that are created get this default user automatically assigned as the creator. This is purely for tagging purposes and there is no functionality tied to it.

Reply:
Added